units/src/amount.rs

```diff
@@ -821,9 +821,17 @@ fn fmt_satoshi_in(
     if total_decimals > 0 {
         write!(f, ".")?;
     }
-    if norm_nb_decimals > 0 {
+    
```

Comment:
In this case you can run either one and it'll be fine, as long as you also rebase on master (since the other formatting line has been fixed and merged in #2683).

But in general you should not format files that are unrelated to your PR. You can do this by running a formatter and then only committing the files that are relevant to your PR.

Or, you could just remove the trailing whitespace which is easy to find with `git grep ' $'`.